Cargo.toml

```diff
@@ -56,23 +58,31 @@ strum_macros = { version = "0.24.3" }
 textwrap = { version = "0.16.0" }
 titlecase = { version = "2.2.1" }
 toml = { version = "0.5.9" }
-update-informer = { version = "0.5.0", default-features = false, features = ["pypi"], optional = true }
 walkdir = { version = "2.3.2" }
+wasm-bindgen = { version = "0.2.83" }
```

Comment:
Can this not be under `target_family = "wasm"`?

Reply:
It can, I moved it there.